docs/guides-agent-app.md

```diff
@@ -0,0 +1,506 @@
+
+- An Aragon DAO to participate as a stakeholder in another DAO.
+
+# Prerequisites
+
+
+
+While there are no formal prerequisites to this guide, it'll be helpful for you to have a basic understanding of both our [Voting](https://wiki.aragon.org/dev/apps/voting/) and [Token Manager](https://wiki.aragon.org/dev/apps/token-manager/) apps. The best way to do that is to go through sections 2.1 and 2.2 of our [User Guide](https://wiki.aragon.org/tutorials/Aragon_User_Guide/index.html#2-templates).
+
+Apart from that this guide should be self-contained.
+
+# Installing Aragon Agent
+
+## 0. Create a Democracy DAO
+
+Before we start, you'll need to head over to [Aragon](https://rinkeby.aragon.org/) and create a new DAO with the [democracy template](https://github.com/aragon/dao-kits/tree/master/kits/democracy).
+
+If you're not sure how to do that, please have a look at  [this section](https://wiki.aragon.org/tutorials/Aragon_User_Guide/#21-create-a-new-democracy-organization) of our  [User Guide](https://wiki.aragon.org/tutorials/Aragon_User_Guide/#21-create-a-new-democracy-organization).
+
+The first thing you'll be asked to do is to [choose the network](https://wiki.aragon.org/tutorials/Aragon_User_Guide/#211-navigate-to-httpsapparagonorg-in-your-web-browser) for your organization. For the purposes of this guide we'll choose the **Ethereum Testnet (Rinkeby)**.
+
+Later on, you'll be asked to [set three parameters](https://wiki.aragon.org/tutorials/Aragon_User_Guide/#214-set-the-parameters-of-your-democracy-organization-then-click-next) for your organization -- the **support**, the **minimum acceptance quorum**, and the **vote duration**.
+
+We'll go with the following (sensible) defaults:
+
+- Support: 100%
+- Min. Quorum: 0%
+- Duration: 168 hours (or 1 week)
+
+## 1. Install aragonCLI
+
+The [aragonCLI](https://hack.aragon.org/docs/cli-intro) (Command Line Interface) is what we use to create, interact with, and develop Aragon apps.
+
+If you haven't done so already, install it from NPM by running the following command:
+
+`npm install -g @aragon/cli`
+
+Hopefully, it installed successfully 😊.
+
+ If not, we recommend you take a quick look at the [installing aragonCLI](/docs/guides-faq.md#installing-aragoncli) section of our [troubleshooting guide](/docs/guides-faq.md#installing-aragonCLI). It should help diagnose and fix the problem 🧐.
+ 
+If that still doesn't fix things 😟, please reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help). We're more than happy to help.
+
+Note that aragonCLI v5.4.0 was the first version to [include support](https://blog.aragon.one/aragon-agent-beta-release/) for Agent.
+
+
+If you're unsure which version of aragonCLI you have, run:
+
+`aragon -v`
+
+If your version number is less than 5.9.5, or if it's been a while since you last installed the CLI, we recommend you reinstall it (by running the `npm install` command above).
```

Comment:
```suggestion
If your version number is less than `5.9.5`, or if it's been a while since you last installed the aragonCLI, we recommend you reinstall it (by running the `npm install` command above).